src/generate.rs

```diff
@@ -74,7 +78,9 @@ pub fn implement(field: &Field, mode: &GenMode, params: &GenParams) -> TokenStre
     let fn_name = Ident::new(
         &format!(
             "{}{}{}{}",
-            if has_prefix_attr(field) && (*mode == GenMode::Get || *mode == GenMode::GetMut) {
+            if has_prefix_attr(field)
+                && (*mode == GenMode::Get || *mode == GenMode::GetCopy || *mode == GenMode::GetMut)
```

Comment:
This is getting a little ugly, maybe add a `prefix()` method to `GenMode` that does the right thing?

Reply:
@djc resolved